assets/js/edit.js

```diff
@@ -13,7 +13,10 @@ const {
 	PanelRow,
 	SelectControl,
 	TextControl,
+	RadioControl,
 } = wp.components;
+const Spacer = wp.components.__experimentalSpacer;
```

Comment:
I am concerned with a few experimental components here. Could it broke things up in the future when the component will be released and experimental support drop? Does using the default `<TextControl>` component have a big disadvantage?

Anyway, if we keep these experimental components, need to bump "Requires at least" WordPress version in the plugin readme.txt. We currently declare support of all range of WordPress core from 4.6 and it looks like the block isn't compatible with earlier Block Editor versions.

I've tested in WP 5.6 — getting error message when adding the block.

<img width="653" alt="Screenshot 2022-06-15 at 12 09 33" src="https://user-images.githubusercontent.com/288381/173789904-9462e9b1-eabe-4277-8b30-675b8c356ebc.png">

Reply:
@cadic You're absolutely right. Experimental controls are not suitable for backward compatibility. I replaced those experimental controls with "TextControl" and also in classic editor, I replaced the number inputs with text inputs.